frontend/package.json

```diff
@@ -14,13 +14,18 @@
     "prepare": "cd .. && husky install .husky"
   },
   "dependencies": {
+    "axios": "^1.6.7",
+    "dropzone": "^6.0.0-beta.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
+    "react-modal": "^3.16.1",
```

Comment:
Do we need these new packages? I didn't run npm install and the code works just fine. If its needed for the file up load we'll keep it. If we don't need them let's remove them.